examples/record-optional-fields/record_optional_fields.bal

```diff
@@ -17,7 +17,7 @@ type Person record {
 public function main() {
     // The `gender` must be provided when creating the record since `Gender` is not a type with an implicit initial value
     // and an explicit default value is not specified for it in the record descriptor.
-    Person p = {gender: "male"};
```

Comment:
The description here must also change now. Maybe to something like the following: "The `gender` and `lname` fields must be provided when creating the record since explicit default values are not specified for these in the record descriptor."

Reply:
I think gender is different case than lname. Thus, explained separately.

Reply:
No, that doesn't apply now. Previously, `lname` had an implicit initial value since it is a string, making it an optional field in essence. But now, since we don't have implicit initial values, `lname` is also a required field.

Reply:
done